src/main/java/dk/kb/image/util/ImageAccessValidation.java

```diff
@@ -30,12 +33,108 @@ public static enum ACCESS_TYPE {
+            List<Integer> JTL, List<Integer> PTL, String CVT) {
+        
+        //FIF and CVT allowed. WID and HEI checked below
+        if (  (RGN != null && RGN.size() !=0) || QLT != null || CNT != null ||  (ROT != null && !"0".equals(ROT)) || GAM != null || CMP != null || PFL != null || INV != null || COL != null  || (JTL != null && JTL.size() >0 ) || (PTL != null && PTL.size() >0 )) {            
+            log.debug("Fullsize for IIP request since a custom parameter was defined. Identifier={}", FIF);
+            return false;
+        }
+        else if (WID == null || HEI == null) {
+            log.debug("Fullsize for IIP request since size parameter was not defined. Identifier={}", FIF);            
+            return false;
+        }
+        else if ( (WID != null && WID > ServiceConfig.getConfig().getInteger("thumbnail.max_width")) || (HEI != null && HEI > ServiceConfig.getConfig().getInteger("thumbnail.max_height")) ) {
+            log.debug("Fullsize for IIP request since size parameter was over thumbnail size. Identifier={}, width={}, height={}", FIF, WID, HEI);
+            return false;
+        }
+        
+        return true;
+    }
+
+    
+    /** Is request classified as a thumbnail or fullsize for IIIF requests.
+     * 
+     * This implementation is very conservative and will determine thumbnail also if most non size-parameters are defined.
+     * It is better to be conservative and later loosen up than giving too much control over thumbnail extraction.
+     * 
+     * Will be full size if any other parameters than FIF and CVT is defined. Also WID and HEI must be below a defined limit in the configuration or it will also be fullsize.
+
+     * For a full description of all arguments see method:  {@link AccessApiServiceImpl#ifffImageRequest() IIUF-parameters} 
+     * @return true if image request is classified as thumbnail request. Else false.    
+     */
+    public static boolean isThumbnailIIIF(String identifier, String region, String size, String rotation, String quality, String format) {
+         if (!"full".equals(region)  ||  !rotation.equals("0") || !quality.equals("default")) {                     
+             log.debug("Fullsize for IIIF request since a custom parameter was defined. Identifier={}", identifier);             
+             return false;                
+         }
+         else if (size == null) {
+             log.debug("Fullsize for IIIF request since size parameter was not defined Identifier={}", identifier);
+             return false;
+         }                           
+         
+            //Only allow size as "w,h" parameter. Etc. "100,100". Value is default 'max' when requesting the full image.
+        
+         Matcher matcher = IIIF_SIZE_PATTERN.matcher(size);
+         if (!matcher.find()) {
+        	 return false;
```

Comment:
Please log what went wrong here, preferably with the `size` parameter so that the developers can be made aware of what callers are attempting to request.